tools/rosbash/rosfish

```diff
@@ -279,6 +279,26 @@ function roscp
     cp $arg $argv[3]
 end
 
+function roscat
+    set -l arg
+    if test (count $argv) = 0
```

Comment:
This condition hasn't been updated. Have you tried this case as it is? Does this work with fish?